oscli/build/Dockerfile.newton

```diff
@@ -0,0 +1,59 @@
+FROM ubuntu:16.04
+
+MAINTAINER Christian Schnidrig
+
+RUN apt-get update && \
+    DEBIAN_FRONTEND=noninteractive apt-get install -y \
+        python-dev \
+        python-pip \
+        software-properties-common \
+        ubuntu-cloud-keyring \
+        sudo && \
```

Comment:
please add git, vim and less
I need git for managing ini files in scripts using openstack cli.
vim and less are always handy to have in a shell.